docs/params-file-grammar.md

```diff
@@ -1,17 +1,17 @@
 program -> statement* EOF 
 statement -> 
   usingDecl |
-  parameterDecl |
+  paramDecl |
   NL
 
 usingDecl ->
-  "using" stringLiteral NL
+  "using" (stringLiteral | "none") NL
 
+paramDecl ->
+  "param" IDENTIFIER(name) "=" literalValue NL
 
-  importDecl -> decorator* "import" IDENTIFIER(providerName) "as" IDENTIFIER(aliasName) object? NL
```

Comment:
Imports are allowed in params files, but this statement doesn't look right. Could you copy the import statement section from the main grammar file? Specifically these lines:

https://github.com/Azure/bicep/blob/126d295799618f6a3b1219b7ad27fd9656095703/docs/grammar.md?plain=1#L27-L41

Reply:
It is unfortunate that neither of the documentation pages I read included that information. I will migrate the import declaration from the main grammar.

https://learn.microsoft.com/en-us/azure/azure-resource-manager/bicep/bicep-import
https://learn.microsoft.com/en-us/azure/azure-resource-manager/bicep/parameter-files?tabs=Bicep